### PR TITLE
Remove spark 2.4 connector from the release

### PR DESCRIPTION
**Pull request by isha97**

### CODE REVIEW

File: README-template.md [hunk 1]
```diff
@@ -68,12 +68,11 @@ The latest version of the connector is publicly available in the following links
 | Spark 3.3  | `gs://spark-lib/bigquery/spark-3.3-bigquery-${next-release-tag}.jar`([HTTP link](https://storage.googleapis.com/spark-lib/bigquery/spark-3.3-bigquery-${next-release-tag}.jar))                                        |
 | Spark 3.2  | `gs://spark-lib/bigquery/spark-3.2-bigquery-${next-release-tag}.jar`([HTTP link](https://storage.googleapis.com/spark-lib/bigquery/spark-3.2-bigquery-${next-release-tag}.jar))                                        |
 | Spark 3.1  | `gs://spark-lib/bigquery/spark-3.1-bigquery-${next-release-tag}.jar`([HTTP link](https://storage.googleapis.com/spark-lib/bigquery/spark-3.1-bigquery-${next-release-tag}.jar))                                        |
-| Spark 2.4  | `gs://spark-lib/bigquery/spark-2.4-bigquery-${next-release-tag}.jar`([HTTP link](https://storage.googleapis.com/spark-lib/bigquery/spark-2.4-bigquery-${next-release-tag}.jar))                                        |
```
Comment (davidrabinowitz) [inline README-template.md:71]: The line should remain, with a fixed `0.37.0` version (like Scala 2.11)

File: README-template.md [hunk 2]
```diff
@@ -87,7 +86,6 @@ below.
 | spark-3.3-bigquery                    |         |         |         |         |         | &check; | &check; | &check; |
 | spark-3.2-bigquery                    |         |         |         |         | &check; | &check; | &check; | &check; |
 | spark-3.1-bigquery                    |         |         |         | &check; | &check; | &check; | &check; | &check; |
-| spark-2.4-bigquery                    |         | &check; |         |         |         |         |         |         |
```
Comment (davidrabinowitz) [inline README-template.md:90]: Keep the line

File: README-template.md [hunk 3]
```diff
@@ -118,7 +115,6 @@ repository. It can be used using the `--packages` option or the
 | Spark 3.3  | `com.google.cloud.spark:spark-3.3-bigquery:${next-release-tag}`                    |
 | Spark 3.2  | `com.google.cloud.spark:spark-3.2-bigquery:${next-release-tag}`                    |
 | Spark 3.1  | `com.google.cloud.spark:spark-3.1-bigquery:${next-release-tag}`                    |
-| Spark 2.4  | `com.google.cloud.spark:spark-2.4-bigquery:${next-release-tag}`                    |
```
Comment (davidrabinowitz) [inline README-template.md:121]: The line should remain, with a fixed `0.37.0` version (like Scala 2.11)

File: README-template.md [hunk 4]
```diff
@@ -68,12 +68,11 @@ The latest version of the connector is publicly available in the following links
...
-| Spark 2.4  | `gs://spark-lib/bigquery/spark-2.4-bigquery-${next-release-tag}.jar`([HTTP link](https://storage.googleapis.com/spark-lib/bigquery/spark-2.4-bigquery-${next-release-tag}.jar))                                        |
 | Scala 2.13 | `gs://spark-lib/bigquery/spark-bigquery-with-dependencies_2.13-${next-release-tag}.jar` ([HTTP link](https://storage.googleapis.com/spark-lib/bigquery/spark-bigquery-with-dependencies_2.13-${next-release-tag}.jar)) |
 | Scala 2.12 | `gs://spark-lib/bigquery/spark-bigquery-with-dependencies_2.12-${next-release-tag}.jar` ([HTTP link](https://storage.googleapis.com/spark-lib/bigquery/spark-bigquery-with-dependencies_2.12-${next-release-tag}.jar)) |
 | Scala 2.11 | `gs://spark-lib/bigquery/spark-bigquery-with-dependencies_2.11-0.29.0.jar` ([HTTP link](https://storage.googleapis.com/spark-lib/bigquery/spark-bigquery-with-dependencies_2.11-0.29.0.jar))                           |
 
-The first four versions are Java based connectors targeting Spark 2.4/3.1/3.2/3.3 of all Scala versions built on the new
+The first five versions are Java based connectors targeting Spark 2.4/3.1/3.2/3.3/3.4/3.5 of all Scala versions built on the new
```
Comment (davidrabinowitz) [inline README-template.md:75]: *six*

**Review by davidrabinowitz — CHANGES_REQUESTED** [review]

File: README-template.md (hunk 1 above)
Comment (isha97) [reply README-template.md:71]: Done

File: README-template.md (hunk 3 above)
Comment (isha97) [reply README-template.md:121]: Done

**Review by davidrabinowitz — APPROVED** [review]

**Pull request merged**